MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/model/usermodel/Mapviews.java

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.mapboxandroiddemo.model.usermodel;
+
+/**
```

Comment:
@langsmith 
👀 There are still classes with file headers: `Mapviews`, `Plan`, `Services`, `UserDirections` and `UserResponse`.